### PR TITLE
[DFFA-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by Badabee**
Screenshot 2021-04-21 at 21 17 43

### CODE REVIEW

File: submissions/bettyBada/html-forms/signup.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+let firstName = document.getElementById("firstName");
+let lastName = document.getElementById("lastName");
+let otherName = document.getElementById("otherName");
+let phoneNumber = document.getElementById("number");
+let email = document.getElementById('email');
+let password = document.getElementById("password");
+let confirmPassword = document.getElementById("confirmPassword");
+
+function firstNameValidation(field, event) {
+    let regExp = /\W/;
```
Comment (Ireti203) [inline submissions/bettyBada/html-forms/signup.js:11]: Does this regex expression cover all unwanted characters a user might use?

**Review by Ireti203 — CHANGES_REQUESTED** [review]
your js file should be in the javascript folder, not html-forms. Kindly update that.

File: submissions/bettyBada/html-forms/signup.js (hunk 1 above)
Comment (Badabee) [reply submissions/bettyBada/html-forms/signup.js:11]: Yes, it covers it.
Screenshot 2021-04-22 at 17 05 12

**Comment by Ireti203** [comment]
After changing the directory of the javascript file, you should also update the script src in the html file

**Comment by Badabee** [comment]
Script src in the HTML file has been updated. Kindly review.

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**